From 77a2fb7ae065f914c6c478d974cd60349eeca817 Mon Sep 17 00:00:00 2001
From: Igor Russkikh <irusskik@redhat.com>
Date: Fri, 6 Nov 2020 18:38:36 -0500
Subject: [PATCH 139/139] [netdrv] net: atlantic: fix build when object tree is
 separate

Message-id: <1604687916-15087-140-git-send-email-irusskik@redhat.com>
Patchwork-id: 338560
Patchwork-instance: patchwork
O-Subject: [RHEL8.4 BZ 1857861 139/139] net: atlantic: fix build when object tree is separate
Bugzilla: 1857861
RH-Acked-by: David Arcari <darcari@redhat.com>
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: http://bugzilla.redhat.com/1857861

commit 059432495e209279bae12db3d2b0bc8c8fe987bb
Author: Igor Russkikh <irusskikh@marvell.com>
Date:   Fri Sep 25 23:27:35 2020 +0300

    net: atlantic: fix build when object tree is separate

    Driver subfolder files refer parent folder includes in an
    absolute manner.

    Makefile contains a -I for this, but apparently that does not
    work if object tree is separated.

    Adding srctree to fix that.

    Signed-off-by: Igor Russkikh <irusskikh@marvell.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Igor Russkikh <irusskik@redhat.com>

Cc: David Arcari <darcari@redhat.com>
Cc: Igor Russkikh <irusskik@redhat.com>
Signed-off-by: Jan Stancek <jstancek@redhat.com>
---
 drivers/net/ethernet/aquantia/atlantic/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/net/ethernet/aquantia/atlantic/Makefile b/drivers/net/ethernet/aquantia/atlantic/Makefile
index 2f5ec1a60be0..6059711823e6 100644
--- a/drivers/net/ethernet/aquantia/atlantic/Makefile
+++ b/drivers/net/ethernet/aquantia/atlantic/Makefile
@@ -8,6 +8,8 @@
 
 obj-$(CONFIG_AQTION) += atlantic.o
 
+ccflags-y += -I$(srctree)/$(src)
+
 atlantic-objs := aq_main.o \
 	aq_nic.o \
 	aq_pci_func.o \
-- 
2.13.6

